### PR TITLE
f-header@10.2.1 - Rename `whiteSeamless` theme, change the background of the theme

### PR DESCRIPTION
**Pull request by 1itvinka**
### Changed
- Background of the "whiteSeamless" theme
- Renamed "disappearingWhite" into "whiteSeamless"

---

The background of the header on Search on wide screens should be white.

Screenshot 2022-07-19 at 16 18 37

---

## UI Review Checks

- [x] README and/or UI Documentation has been [created|updated]
- [ ] Unit tests have been [created|updated]
- [x] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [x] Chrome (latest)

### CODE REVIEW

**Review by DamianMullins — DISMISSED** [review]

File: packages/components/organisms/f-header/README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ The props that can be defined are as follows:
...
-| headerBackgroundTheme     | `String`      | `white` | Sets the background theme for the header component.<br><br>When set to `white` the header will be white with the default logo colour.<br>When set to `transparent` the header will be transparent with a white logo.<br>When set to `highlight` the header will use the primary brand colour as the background colour with a white logo.<br>When set to `disappearingWhite` the header will be transparent with with the default logo colour on wide screens, and white with box-shadow at the bottom of the header on narrow screens.  |
+| headerBackgroundTheme     | `String`      | `white` | Sets the background theme for the header component.<br><br>When set to `white` the header will be white with the default logo colour.<br>When set to `transparent` the header will be transparent with a white logo.<br>When set to `highlight` the header will use the primary brand colour as the background colour with a white logo.<br>When set to `disappearingWhite` the header will be white with no box-shadow at the bottom and with the default logo colour on wide screens, and white with box-shadow at the bottom of the header on narrow screens.  |
```
Comment (jamieomaguire) [inline packages/components/organisms/f-header/README.md]: Is the name `disappearingWhite` still appropriate if it's no longer transparent on wide screens?

Comment (1itvinka) [reply packages/components/organisms/f-header/README.md]: good question! Maybe "plainWhite"? The only difference right now is that there's no `box-shadow` at the bottom. @ashleynolan what do you think?

Comment (ashleynolan) [reply packages/components/organisms/f-header/README.md]: I'd probably rename it – maybe something like `whiteSeamless`, as it's intended to merge the header straight into the page content?

Comment (ashleynolan) [reply packages/components/organisms/f-header/README.md]: The alternative would be just to make it a separate prop which controls whether there should be a `boxShadow` or not, which may be more appropriate? (if that happens on narrow and wide screens)

Comment (1itvinka) [reply packages/components/organisms/f-header/README.md]: it happens only on wide screens. There's a `boxShadow` on narrow screens.

Comment (ashleynolan) [reply packages/components/organisms/f-header/README.md]: Ok, maybe keep it as a theme then in that case and just call it `whiteSeamless`

**Review by oliversweb — APPROVED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Pull request merged**